utils/index.js

```diff
@@ -8,17 +8,17 @@ const path = require('path');
 const DEFAULT_VERSION = {
   major: 2,
   minor: 0,
-  patch: 4,
+  patch: 4
 };
 
 exports.isEjected = fs.pathExistsSync(path.join(process.cwd(), 'config/paths.js'))
 
-exports.getReactScriptsVersion = function getReactScriptsVersion(cliVersion) {
+exports.getReactScriptsVersion = function (cliVersion) {
   if (cliVersion) {
     const versions = {
       major: Number(semver.major(cliVersion)),
       minor: Number(semver.minor(cliVersion)),
-      patch: Number(semver.patch(cliVersion)),
+      patch: Number(semver.patch(cliVersion))
```

Comment:
Your IDE seems to be removing the trailing comma, breaking the code styles and adding noise to the commit.

Reply:
Don't worry about it, as I'll work on the file again I'll change them myself.